dev-run.bash

```diff
@@ -11,6 +11,9 @@ function main {(
     export EXECUTOR_FILE_PATH=$(ls $(pwd)/cassandra-executor/target/cassandra-executor-*-jar-with-dependencies.jar)
     export JDK_FILE_PATH=$(pwd)/target/framework-package/jdk.tar.gz
     export CASSANDRA_FILE_PATH=$(pwd)/target/framework-package/cassandra.tar.gz
+    if [ -z $MESOS_NATIVE_JAVA_LIBRARY ] ; then
+        export MESOS_NATIVE_JAVA_LIBRARY=`pwd`/../mesos/build/src/.libs/libmesos.dylib
```

Comment:
This looks environment specific, is it safe to use `pwd` here? (I'm pretty sure this doesn't map to linux install paths)